sklearn/linear_model/logistic.py

```diff
@@ -1208,10 +1208,10 @@ def fit(self, X, y, sample_weight=None):
             raise ValueError("Tolerance for stopping criteria must be "
                              "positive; got (tol=%r)" % self.tol)
 
-        if self.solver in ['newton-cg']:
-            _dtype = [np.float64, np.float32]
-        else:
+        if self.solver in ['lbfgs', 'liblinear']:
```

Comment:
It would be good to document in the Notes section of the object what solvers can handle what dtype.

Reply:
Done!

Reply:
Although I could improve the wording…